src/components/AssetSearch/AssetList.tsx

```diff
@@ -84,7 +84,7 @@ export const AssetList: FC<AssetListProps> = ({
           itemCount={assets.length}
           ref={tokenListRef}
           className='token-list'
-          overscanCount={3}
+          overscanCount={1}
```

Comment:
this is deliberate, should improve perf on mobiles without affecting anything on desktop since desktop renders so fast